DeviceInfo/doc/DeviceInfoPlugin.md

```diff
@@ -73,21 +73,92 @@ The table below lists configuration options of the plugin.
 | callsign | string | Plugin instance name (default: *DeviceInfo*) |
 | classname | string | Class name: *DeviceInfo* |
 | locator | string | Library name: *libWPEFrameworkDeviceInfo.so* |
-| autostart | boolean | Determines if the plugin is to be started automatically along with the framework |
+| autostart | boolean | Determines if the plugin shall be started automatically along with the framework |
+| hdr | boolean | Does the device support HDR (true or false) |
+| atmos | boolean | Does the device support Dolby Atmos (true or false) |
+| cec | boolean | Does the device support HDMI CEC (true or false) |
+| hdcp | string | HDCP version supported by the device (unavailable, hdcp_14, hdcp_20, hdcp_21, hdcp_22) |
+| audio | array | Supported audio outputs |
+| audio[#] | string | Audio output (other, rf_modulator, analog, spdif, hdmi, displayport) |
+| video | array | Supported video outputs |
+| video[#] | string | Video output (other, rf_modulator, composite, svideo, component, scart_rgb, hdmi, displayport) |
+| resolution | array | Supported output resolutions |
+| resolution[#] | string | Output resolution (unknown, 480i, 480p, 576i, 576p, 720p, 1080i, 1080p, 2160p30, 2160p60, 4320p30, 4320p60) |
 
 <a name="head.Properties"></a>
 # Properties
 
-The following properties are provided by the DeviceInfo plugin:
+The following properties are provided by the DeviceInfo interface:
 
 DeviceInfo interface properties:
 
 | Property | Description |
 | :-------- | :-------- |
+| [stbcapabilities](#property.stbcapabilities) <sup>RO</sup> | Capabilities of the STB |
```

Comment:
If the proeprty is stbcapabilities, could we change it to capabilities, method already reflect that tjis is on the DeviceInfo and not on the DisplayInfo